chord_metadata_service/chord/views_search.py

```diff
@@ -88,7 +79,7 @@ def phenopacket_dataset_summary(dataset):
                 "sex": {k: individuals_sex.get(k, 0) for k in (s[0] for s in Individual.SEX)},
                 "karyotypic_sex": {k: individuals_k_sex.get(k, 0) for k in (s[0] for s in Individual.KARYOTYPIC_SEX)},
                 "taxonomy": queryset_stats_for_field(phenopacket_qs, "subject__taxonomy__label"),
-                "age": get_field_bins(phenopacket_qs, "subject__age_numeric", 10),
+                "date_of_birth": phenopacket_qs.values("subject__date_of_birth")
```

Comment:
we should not reveal any date of birth in the dataset summary! especially unbinned, but even binned this is not data that should be exposed in a summary endpoint.